### ISSUE TITLE
Migrate to .NET Standard

### ISSUE THREAD

**Issue by DylanFPS** [Ultz-Limited/SharpDB#19]

**Comment by maikebing** [comment]
Has this work been started yet?

**Comment by DylanFPS** [comment]
No, not yet, as we're fairly busy at the moment. If all goes to plan, though, work on SharpDB 1.0 will start next week ;)

Don't worry, this is one of our top priorities.

**Comment by maikebing** [comment]
I'm trying to do it, and I'll submit a PR later!

**Comment by DylanFPS** [comment]
Okay, good luck! :)
